src/main/java/com/dropbox/core/android/AuthActivity.java

```diff
@@ -581,4 +694,46 @@ private String createStateNonce() {
         }
         return sb.toString();
     }
+
+    private String createPKCEStateNonce() {
+        return String.format("oauth2code:%s:%s:%s",
```

Comment:
We should pass in a locale to be extra safe.

`String.format("oauth2code:%s:%s:%s",` -> `String.format(Locale.US, "oauth2code:%s:%s:%s",`

Reply:
Will change!